### ISSUE TITLE
[Accessibility]: Notepad: Possibly unclear relation between arrow selection behavior and tab/focus behavior

### ISSUE THREAD

**Issue by fkling** [sourcegraph/sourcegraph#35165]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34198

### Problem description

In order to select an item with a keyboard, it needs to be focused (via <kbd>Tab</kbd>) and then can be selected by pressing <kbd>Space</kbd>. This is also how multiple entries can be selected (<kbd>Tab</kbd> to entry and press <kbd>Meta</kbd> + <kbd>Space</kbd>). 
But selecting an item with <kbd>&uarr;</kbd>/<kbd>&darr;</kbd> keys does not move focus.

Selecting a range items can only be done via <kbd>Shift</kbd>+<kbd>&uarr;</kbd>/<kbd>&darr;</kbd> and doesn't move focus either.

### Expected behavior

It's actually not clear to me how focus should change wrt to arrow up/down keys.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by limitedmage** [comment]
@almeidapaulooliveira Can you take a look at this issue and suggest the right way to do this keyboard interaction?

**Comment by limitedmage** [comment]
List items in notepad should be focused when using arrow keys
